frontend/src/core/loaders/components/WaveLoader.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import WaveLoader, { WaveLoader as WaveLoaderBase } from './WaveLoader';
+
+import { createReduxStore } from '../../../test/store';
+import { shallowUntilTarget } from '../../../test/utils';
+
+describe('<WaveLoader>', () => {
+    it('render the loader before l10n bundle will be fetched', () => {
```

Comment:
Nit, these verbs should be 3rd person (`renders`), the goal being to read them as a sentence (`it renders the loader`).